pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -14,13 +14,21 @@
 , python3Packages
 }:
 
+let
+  version = "unstable-2023-06-30";
```

Comment:
IMO the lack of stable releases is (almost) never good for Nixpkgs packaging but I guess we don't even have to ask upstream as it probably makes little sense for such a repository.

Reply:
oh, i dislike this just on the basis of the version string being ugly

but yea, with this repo hosting several things, having them declare versions seems like more trouble than it's worth